lms/views/lti/deep_linking.py

```diff
@@ -179,22 +185,19 @@ def file_picker_to_form_fields_v11(self):
                         {
                             "@type": "LtiLinkItem",
                             "mediaType": "application/vnd.ims.lti.v1.ltilink",
-                            "url": url,
+                            "url": self.misc_plugin.get_deeplinking_launch_url(
+                                self.request, assignment_configuration
+                            ),
+                            "custom": assignment_configuration,
                         },
                     ],
                 }
             )
         }
 
     @staticmethod
-    def _get_content_url(request):
-        """
-        Translate content information from the frontend to a launch URL.
-
-        We submit the content information to the LMS as an URL pointing to our
-        `lti_launches` endpoint with any information required to identity
-        the content as query parameters.
-        """
+    def _get_assignment_configuration(request) -> dict:
```

Comment:
This method changes from both getting the config from the FE and encoding it in an URL to just the the config from the FE.